build.gradle

```diff
@@ -582,8 +582,10 @@ dependencies {
     compile files('lib/randomcutforest-core-2.0.1.jar')
 
     // used for serializing/deserializing rcf models.
-    compile group: 'io.protostuff', name: 'protostuff-core', version: '1.7.4'
-    compile group: 'io.protostuff', name: 'protostuff-runtime', version: '1.7.4'
+    compile files('lib/protostuff-api-1.8.0-SNAPSHOT.jar')
```

Comment:
Do you know when protostuff team plan to publish 1.8.0 to maven? We can move to public maven once it's ready.

Reply:
I don't. asked in their github repo, noboy replied.

Reply:
I couldn't find a link for the ask, opened https://github.com/protostuff/protostuff/issues/327.